### DISCUSSION TITLE
Struggle to detect car at night - Frigate Plus Solution?

### DISCUSSION THREAD

**Discussion by JRascagneres** [blakeblackshear/frigate#13773] [Ask A Question]
Hi all,

I have a camera mounted at the top of my garage and it looks down a little onto my car. In the day it seems to detect it at ~70% positive but at night (B+W), especially with rain it struggles, usually dropping to 50% and it causes it to detect and un-detect it a lot, triggering many events. 

Would a trained model with frigate plus help this? If so how would I go about doing this?

Thanks!

**Comment by NickM-27** [comment]
yes, in my experience frigate+ models are much better at night. Which makes sense since there are no IR images in the COCO dataset.

See https://docs.frigate.video/plus/first_model

**Reply by JRascagneres** [reply]
Thanks. I don't have any 'false positives' for this situation. Would it be fine to omit these?

**Reply by NickM-27** [reply]
if you have false positives you submit them, if you don't have any then no reason to worry. Please be sure to update the filters like the next steps tell you to after getting your model

**Reply by JRascagneres** [reply]
Great thanks. Might sign up

**Reply by JRascagneres** [reply]
Can I specify the model for just one camera? Or am I going to need to make car uploads for every camera?

**Reply by NickM-27** [reply]
No, but technically you can try just uploading images from one camera and see how it works and tune from there
